compiler/noirc_frontend/src/parser/parser/types.rs

```diff
@@ -27,16 +27,7 @@ pub(super) fn parse_type_inner<'a>(
         int_type(),
         bool_type(),
         string_type(),
-        expr_type(),
-        struct_definition_type(),
-        trait_constraint_type(),
-        trait_definition_type(),
-        trait_impl_type(),
-        function_definition_type(),
-        module_type(),
-        top_level_item_type(),
-        type_of_quoted_types(),
-        quoted_type(),
+        comptime_type(),
```

Comment:
I had to extract this to another function because the `Choice` trait was up to a certain tuple length and with one more we exceeded it :-)